unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -689,6 +689,8 @@ mod tests {
     #[test_case("user_id in [1, 3, 6]", true)]
     #[test_case("user_id in [1, 3, 5]", false)]
     #[test_case("user_id not_in [1, 3, 5]", true)]
+    #[test_case("user_id in [\"dfsfsd\"]", false)]
+    #[test_case("user_id not_in [\"dfsfsd\"]", true)]
```

Comment:
Just a minor detail: with this is not clear why `user_id not_in [\"dfsfsd\"]` should be true without looking at the body of the test. I don't know if it's worth adding the `6` parameter from where you're taking the context. 

Something like:
1. scenario: given the context for user id 6
2. when: user_id in [\"dfsfsd\"]
3. then: should be false

Reply:
> Just a minor detail: with this is not clear why user_id not_in [\"dfsfsd\"] should be true without looking at the body of the test. I don't know if it's worth adding the 6 parameter from where you're taking the context.

Ah. Yeah fair enough. I've gotten too used to reading these. Probably not a thing I want to do in this PR but a big refactor of the test names might be worthwhile now that I'm no longer the only person poking

> Also, I noticed that Rust files tend to be really large due to embedded tests

Yeah, you get used to it over time. You start to look for the mod test and mark that in your mind as the relevant test code. Takes a bit of time but I prefer it now

Reply:
> Yeah, you get used to it over time. You start to look for the mod test and mark that in your mind as the relevant test code. Takes a bit of time but I prefer it now

Challenge accepted :rofl: